conjur/wrapper/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""
+wrapper
+
+This module contains wrapper classes for third party libraries
+"""
```

Comment:
imports?

Reply:
Added for ArgumentWrapper.
But HttpWrapper don't have a class so i won't put it here